src/titlepane/tests/functional/TitlePane.ts

```diff
@@ -0,0 +1,44 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+
+function getPage(remote: any) {
+	return remote
+		.get('http://localhost:9000/_build/common/example/?module=titlepane')
+		.setFindTimeout(5000);
+}
+
+const DELAY = 400;
+
+registerSuite({
+	name: 'TitlePane',
+
+	'Should be fully visible when `open`'(this: any) {
+		return getPage(this.remote)
+			.findByCssSelector('#titlePane2 > div > :last-child')
```

Comment:
can you please import the `css` file for `titlepane` and use that to target nodes in `findByCssSelector`.